src/ResourceManagement/Compute/Microsoft.Azure.Management.Compute/Generated/ContainerServicesOperations.cs

```diff
@@ -42,17 +42,198 @@ internal ContainerServiceOperations(ComputeManagementClient client)
         public ComputeManagementClient Client { get; private set; }
 
         /// <summary>
-        /// The operation to create or update a container service.
+        /// Gets a list of container services in the specified subscription.
         /// </summary>
+        /// <remarks>
+        /// Gets a list of container services in the specified subscription. The
+        /// operation returns properties of each container service including state,
+        /// orchestrator, number of masters and agents, and FQDNs of masters and
+        /// agents.
+        /// </remarks>
+        /// <param name='customHeaders'>
+        /// Headers that will be added to request.
+        /// </param>
+        /// <param name='cancellationToken'>
+        /// The cancellation token.
+        /// </param>
+        /// <exception cref="Microsoft.Rest.Azure.CloudException">
+        /// Thrown when the operation returned an invalid status code
+        /// </exception>
+        /// <exception cref="Microsoft.Rest.SerializationException">
+        /// Thrown when unable to deserialize the response
+        /// </exception>
+        /// <exception cref="Microsoft.Rest.ValidationException">
+        /// Thrown when a required parameter is null
+        /// </exception>
+        /// <return>
+        /// A response object containing the response body and response headers.
+        /// </return>
+        public async System.Threading.Tasks.Task<Microsoft.Rest.Azure.AzureOperationResponse<Microsoft.Rest.Azure.IPage<ContainerService>>> ListWithHttpMessagesAsync(System.Collections.Generic.Dictionary<string, System.Collections.Generic.List<string>> customHeaders = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken))
```

Comment:
How can I most easily discover the latest available working nightly? Since the other RPs seem to be on nightly revs from December?